etna/whatson/models.py

```diff
@@ -532,10 +547,19 @@ class EventPage(ArticleTagMixin, TopicalPageMixin, BasePageWithIntro):
     venue_space_name = models.CharField(
         max_length=255,
         verbose_name=_("venue space name"),
```

Comment:
As per Ashley's remarks, ‘Venue space name’ should not be mandatory for Hybrid or In person events. But this is a specialised case so I have removed the nullability of this field and altered code in the clean() function below.